models/comments.models.js

```diff
@@ -0,0 +1,19 @@
+const db = require("../db/connection");
+const { checkCommentId } = require("../utils/checkCommentId");
+
+exports.deleteComment = (id) => {
+  return checkCommentId(id)
+    .then(() => {
+      return db.query(
+        `
+      DELETE FROM comments 
+      WHERE comment_id = $1
+      RETURNING *;`,
+        [id]
+      );
+    })
+    .then(({ rows }) => {
+      console.log(rows[0]);
```

Comment:
delete console logs

Reply:
Thanks for the feedback @pippaamy , did not even realise I was sending stuff on the deletes. I will make changes from both the PRs after I complete the current task I'm on :)